websites/V/Voirdrama/metadata.json

```diff
@@ -0,0 +1,24 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.10",
+	"author": {
+		"id": "709704536821858304",
+		"name": "cyadine"
+	},
+	"service": "Voirdrama",
+	"description": {
+		"en": "Voiranime is a streaming website where you can watch Asian drama."
```

Comment:
This description seems to describe the wrong service. Try using an official description from the service itself